praw/models/reddit/modmail.py

```diff
@@ -75,6 +75,24 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
     def _info_path(self):
         return API_PATH['modmail_conversation'].format(id=self.id)
 
+    def reply(self, body, author_hidden=False, internal=False):
+        """Reply to the conversation.
+
+        :param body: The markdown formatted content for a message.
+        :param author_hidden: When True, author is hidden from non-moderators.
+        :param internal: When True, message is hidden from non-moderators.
+        :returns: A :class:`~.ModmailMessage` object for the newly created
+            message.
+
+        """
+        data = {'body': body, 'isAuthorHidden': author_hidden,
+                'isInternal': internal}
+        response = self._reddit.post(API_PATH['modmail_conversation']
+                                     .format(id=self.id), data=data)
+        message_data = max(response['messages'].values(),
```

Comment:
I'm guessing that the entire conversation is returned. Is there a more direct index based way to get the new message rather than using max based on date? Not that this is slow, I just think logically it would be simpler to directly grab the most recent message. If I recall correctly there is an attribute which lists the message order, right?